examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+
+# Re-normalize our empty-room projectors, so they are fine after subselection
+raw.info.normalize_proj()
+# Read epochs
+proj = False  # already applied
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                    baseline=(None, 0), preload=True, proj=proj,
+                    reject=dict(grad=4000e-13, mag=4e-12, eog=150e-6))
+evoked = epochs.average()
+
+# Read regularized noise covariance and compute regularized data covariance
+noise_cov = mne.compute_covariance(epochs, tmin=tmin, tmax=0, method='shrunk')
+data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
+                                  method='shrunk')
+# Read forward model
+forward = mne.read_forward_solution(fname_fwd)
+
+# Compute weights of free orientation (vector) beamformer with weight
+# normalization (neural activity index, NAI). Providing a noise covariance
+# matrix enables whitening of the data and forward solution. Source orientation
+# is optimized by setting pick_ori to 'max-power'.
+# weight_norm can also be set to 'unit-noise-gain'. Source orientation can also
+# be 'normal' (but only when using a surface-based source space) or None,
+# which computes a vector beamfomer. Note, however, that not all combinations
+# of orientation selection and weight normalization are implemented yet.
+filters = make_lcmv(evoked.info, forward, data_cov, reg=0.05,
+                    noise_cov=noise_cov, pick_ori='max-power',
+                    weight_norm='nai')
+
+# Apply this spatial filter to the evoked data.
+stc = apply_lcmv(evoked, filters, max_ori_out='signed')
+
+# take absolute values for plotting
+stc.data[:, :] = np.abs(stc.data)
+
+# Save result in stc files
+stc.save('lcmv-vol')
+
+# stc.crop(0.0, 0.2)
+# Save result in a 4D nifti file
+mne.save_stc_as_volume('lcmv_inverse.nii.gz', stc,
+                       forward['src'], mri_resolution=True)  # set to true
+
+###############################################################################
+# Load nifti data
+
+# load lcmv inverse
+stc = nib.load('lcmv_inverse.nii.gz')
+
+# select first time volume and output ndarray
+stc = stc.dataobj[:, :, :, 1]
+
+# normalize values
+stc = stc / stc.max()
+
+# load subject brain (Moving)
+t1_fname = data_path + '/subjects/sample/mri/brain.mgz'
+t1_m = nib.load(t1_fname)
+
+# get Moving to world transform
+t1_m_grid2world = t1_m.affine
+
+# output Moving as ndarray
+t1_m = t1_m.dataobj[:, :, :]
+
+# normalize values
+t1_m = t1_m.astype('float') / t1_m.max()
+
+# load fsaverage brain (Static)
+t1_fname = data_path + '/subjects/fsaverage/mri/brain.mgz'
+t1_s = nib.load(t1_fname)
+
+# get Static to world transform
+t1_s_grid2world = t1_s.affine
+
+# output Static as ndarray
+t1_s = t1_s.dataobj[:, :, :]
+
+# normalize values
+t1_s = t1_s.astype('float') / t1_s.max()
+
+##############################################################################
+# Compute affine transformation using mutual information metric
```

Comment:
can you mention why this metric (it's standard, typically least error, etc.)

Reply:
It's best to have one `References` section -- I'd move the `References` to its own section at the bottom